lola.html

```diff
@@ -905,31 +1012,101 @@ <h2>Followup Work</h2>
+        If a Following list was copied, the destination may issue Follow activities.  The destination may also provide 
+        an option to NOT do this in case the user wishes to test out the account and content before committing to 
+        issuing Follow activities from this new location.
       </p>
 
       <p>
-        If a Following list was copied, the destination may issue Follow activities.  The destination may also provide an option to NOT do this in case the user wishes to test out the account and content before committing to issuing Follow activities from this new location.
+        Reporting failed attempts back to the user is also up to the destination server using its own UI and 
+        notification channels.  A destination MAY fail an account transfer for a number of reasons (e.g. quota reached, 
+        access lost) and SHOULD NOT keep an account transfer job going forever, although appropriate timeouts are not 
+        specified here. 
       </p>
 
+
       <h3>Followup Requirements on Source Server</h3>
 
       <h4>Notifying Followers</h4>
 
       <p>
-        The source server SHOULD offer UI for the user to choose to notify followers of a new location.  This is done via the Move Actor FEP mechanism.  Although the destination can also issue a move, it is preferred that the source server do this if it is available.
+        The source server SHOULD offer UI for the user to choose to notify followers of a new location.  This is 
+        done via the Move activity type, defined in 
+        <a href="https://www.w3.org/TR/activitystreams-vocabulary/#dfn-move">ActivityStreams Vocabulary</a> 
+        and described more for use to move Actors in the 
+        <a href="https://codeberg.org/fediverse/fep/src/branch/main/fep/7628/fep-7628.md">Move Actor FEP</a>.
+      </p>
+
+      <p>
+        Load management: it is possible that the source server issuing Move activities could cause many follower
+        agents to issue a great deal of data fetching requests to the destination.  A malicious source server
+        could issue false Move Activities in an attempt to overload the alleged destination server with follow
+        requests and other requests. More discussion of this threat is welcome, but so far this seems self-limiting
+        because a malicious source server would have to have followers to send to a false destination, and
+        would presumably lose its followers and run out of them during this process.
+      </p>
+
+      <p>
+        Open Issue: Can the destination server issue a Move activity if necessary?  It cannot be trusted, because a 
+        malicious actor could issue a Move activity for an account they have no legitimate access to.  It's not enough 
+        to check the source Actor to see if it has a movedTo value, because the malicious actor could specifically 
```

Comment:
I don't follow why it's not enough? The nomadic-style (actor is own root of trust and verification of client-signed properties) is certainly one way of authenticating such requests, and many AP implementations today rely on `rel-me` or `movedTo` properties as authentication here). See silverpill's descriptive FEP:
https://codeberg.org/fediverse/fep/src/branch/main/fep/7628/fep-7628.md